modules/database/export.bal

```diff
@@ -604,3 +592,19 @@ public class exportJob {
     }
 }
 
+# Delete export entry
+#
+# + experimentId - experiment id
+# + exportId - export db id
+# + return - error or empty
+public isolated transactional function deleteExport(int experimentId, int exportId) returns error? {
+    var result = experimentDB->execute(
+        `DELETE FROM ExperimentExport WHERE exportId=${exportId} AND experimentId=${experimentId};`
+    );
+
+    if result is error {
+        return result;
+    } else {
+        return;
+    }
```

Comment:
Just use check and use an `_` as variable name to tell ballerina that you do not need the result.